pkg/api/snapshots.go

```diff
@@ -32,31 +30,22 @@ func (client *Client) ShowSnapshots(snapshotId string, sourceVolumeId string) ([
 	var err error
 	var httpRes *http.Response
 	var response *openapiclient.SnapshotsObject
+	var status *common.ResponseStatus
 
 	// Call the correct OpenAPI Client function
 	if sourceVolumeId != "" {
-		response, httpRes, err = client.apiClient.DefaultApi.ShowSnapshotsVolumeGet(client.Ctx, sourceVolumeId).Execute()
+		response, status, httpRes, err = ExecuteWithFailover(client.apiClient.DefaultApi.ShowSnapshotsVolumeGet(client.Ctx, sourceVolumeId).Execute, client)
```

Comment:
Should Execute() do failover by default and an ExecuteWithoutFailover() be available for the rare exception where you wouldn't want failover?